lib/msal-core/src/utils/Utils.ts

```diff
@@ -1,14 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What about naming this `utils/index.ts`?

Reply:
This file exports the Utils class object, which is why it is named Utils.ts. Do we turn all of the functions into globals?